tests/unit/Assets/Filters/CssMinTest.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Phalcon\Test\Unit\Assets\Filters;
+
+use Phalcon\Assets\Resource;
+use Phalcon\Test\Proxy\Assets\Filters\Cssmin;
+use Phalcon\Test\Module\UnitTest;
+
+/**
+ * \Phalcon\Test\Unit\Assets\Filters\Cssmin.php
+ * Tests the \Phalcon\Assets\Filters\Cssmin component
+ *
+ * @copyright (c) 2011-2016 Phalcon Team
+ * @link      http://www.phalconphp.com
+ * @author    Andres Gutierrez <andres@phalconphp.com>
+ * @author    Nikolaos Dimopoulos <nikos@phalconphp.com>
+ * @package   Phalcon\Test\Unit\Assets\Filters
+ * @group     assets
+ *
+ * The contents of this file are subject to the New BSD License that is
+ * bundled with this package in the file docs/LICENSE.txt
+ *
+ * If you did not receive a copy of the license and are unable to obtain it
+ * through the world-wide-web, please send an email to license@phalconphp.com
+ * so that we can send you a copy immediately.
+ */
+class CssMinTest extends UnitTest
+{
+    /**
+     * Tests cssmin filter with empty parameters
+     *
+     * @author Nikolaos Dimopoulos <nikos@phalconphp.com>
+     * @since  2014-10-10
+     */
+    public function testAssetsFilterCssmin()
+    {
+        $this->specify(
+            "The cssmin filter needs a string parameter",
+            function () {
+                $cssmin = new Cssmin();
+
+                try {
+                    $cssmin->filter(new \stdClass());
```

Comment:
instead of try-catch you can use `throws` in Specify to handle and check exceptions
Currently, If nothing is thrown - this test will pass